purchase_transport_mode/tests/test_purchase_transport_mode.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo.tests import TransactionCase
+
+
+class TestPurchaseTransportMode(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
just found out that in v16 we can use `from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT` to set all mail tracking keys

Reply:
thank you ! 
indeed it will disable the follwing:
```
DISABLED_MAIL_CONTEXT = {
    'tracking_disable': True,
    'mail_create_nolog': True,
    'mail_create_nosubscribe': True,
    'mail_notrack': True,
    'no_reset_password': True,
}
```